tests/common_test.py

```diff
@@ -40,25 +26,6 @@ def test_counter(self):
         c.count("a")
         self.assertEqual("a : 1\nb : 3\n", c.to_str())
 
-    def test_rescale_value(self):
```

Comment:
We can use `numpy.interp()` if something like `rescale_value()` is needed. As with many of the functions in `common.py`, it wasn't used anywhere anyway.